### PR TITLE
Improve overall documentation [skip ci]

### PR DESCRIPTION
**Pull request by arkgil**
Addresses #5

### CODE REVIEW

**Comment by arkgil** [comment]
@mentels here is another one with documentation.

File: lib/fennec.ex [hunk 1]
```diff
@@ -1,6 +1,19 @@
 defmodule Fennec do
   @moduledoc """
-  STUN/TURN server
+  STUN/TURN servers application
```
Comment (mentels) [inline lib/fennec.ex]: Maybe just `STUN/TURN server`?

File: lib/fennec.ex [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+
+  Fennec allows you to start multiple listeners (later called "servers")
+  which react to STUN/TURN messages. The only difference between servers
```
Comment (mentels) [inline lib/fennec.ex]: between the servers.... which a server users....

File: lib/fennec.ex [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+  is a transport protocol and interface/port pair which server uses for
+  listening to STUN packets.
+
+  Each server is independent, i.e. if one of them crashes other should not
```
Comment (mentels) [inline lib/fennec.ex]: the other one should not ... the recommended method... depend on a server's supervisor configuration.

File: lib/fennec.ex [hunk 4]
```diff
@@ -1,6 +1,19 @@
...
+  be affected (note: this applies when starting servers using recommended method -
+  via application's configuration. If you hook up a server to your supervision tree
+  the behaviour will depend on configuration of supervisor of a server).
+
+  Currently only UDP transport is supported. Read more about it in documentation
```
Comment (mentels) [inline lib/fennec.ex]: the documentation

**Pull request merged**